doc/build-windows.md

```diff
@@ -81,27 +81,6 @@ The first step is to install the mingw-w64 cross-compilation tool chain:
 
     sudo apt install g++-mingw-w64-x86-64
```

Comment:
This always installs both variants?

Reply:
Apparently yes: https://packages.ubuntu.com/hirsute/g++-mingw-w64-x86-64

`g++-mingw-w64-x86-64` is an umbrella package that depends on `g++-mingw-w64-x86-64-posix` and `g++-mingw-w64-x86-64-win32`. It does not contain any compiler itself, just some documentation files.